src/lib/utils.spec.ts

```diff
@@ -0,0 +1,17 @@
+import test from 'ava';
+
+import { parseRevisionNumber } from './utils';
+
+test('can parse revision numbers', (t) => {
+  const musselnet = parseRevisionNumber('musselnet-4');
+  t.is(musselnet.toNumber(), 4);
+
+  const numerific = parseRevisionNumber('numers-123-456');
```

Comment:
Should this throw? If not how about these test cases?
- `''`
- `'-'`
- `'hello-'`
- `'hello-123-'`
- `'hello-0123'`
- `'hello-00123'`
- `'hello-1.23'`

Reply:
It should never throw. This is the description of what the SDK does (from cosmos discord)

```
So currently it looks like any chain ID with an ending "-", followed by a non-zero digit, then any number of [0-9] digits will be interpreted as being in "Epoch Format", so for those chains, IBC packets originating from that chain must have their "revision_number" field set to that number.
```